src/vcHotkey.h

```diff
@@ -41,6 +41,13 @@ namespace vcHotkey
   bool IsDown(vcBind key);
   bool IsPressed(vcBind key);
   void GetKeyName(vcBind key, char *pBuffer, uint32_t bufferLen);
+
+  template <size_t N>
+  void GetKeyName(vcBind key, char(&buffer)[N])
```

Comment:
This should probably marked as `inline`, but also, you can just put this on a single line - see `udStringUtil.h` in udCore for examples of how this is done.